src/jsx/components/input_info.jsx

```diff
@@ -71,26 +87,29 @@ class InputInfo extends React.Component {
                 <a onClick={()=>this.saveTheJSON()}>Save JSON</a>
               </li>
               <li className="dropdown-item">
-                <a onClick={()=>this.open()}>View JSON</a>
+                <a onClick={()=>this.open('json')}>View JSON</a>
               </li>
             </ul>
           </div>
         </div>
 
-        <Modal show={this.state.showModal} onHide={()=>this.close()}>
+        <Modal show={Boolean(this.state.showModal)} onHide={()=>this.close()} dialogClassName="my-modal">
           <Modal.Header>
```

Comment:
The bootstrap docs seem to list a `myModal` class but I didn't see it with the hyphen (`my-modal`) like you haver here. Just wanted to make sure this is correct.

Reply:
I agree that `my-modal` is a really generic sounding name.